jtabba/CDW-1
# Nav bar

## Type of change

[x] New feature
<br />
[] Bug fix
<br />
[] Documentation
<br />
[] Code refactor/tech debt
<br />
[] Infrastructure


## Description
This implements a responsive navbar that has a mobile view (drop down)


## Ticket link
[Ticket](https://linear.app/cdw/issue/CDW-43/style-the-navbar)


## Part of a back end implementation?
[x] No
<br />
[] [Yes]()  


## How to test


1. Checkout this branch and run frontend
2. Check nav buttons by navigating between pages
3. Resize window to check repsonsiveness
4. Toggle mobile view in devtools to see dropdown hamburger menu
5. Click hamburger menu to make sure dropdown toggles


## Screenshots

![image](https://github.com/jtabba/CDW-1/assets/12403004/9034ee8b-5d38-4c44-93e8-fa632431818f)
![image](https://github.com/jtabba/CDW-1/assets/12403004/462c528a-a969-4e16-8600-d203c3f20e4a)
![image](https://github.com/jtabba/CDW-1/assets/12403004/3289e31d-0987-4d2d-977f-4c4ae52bbbd4)
